cmeutils/structure.py

```diff
@@ -130,18 +129,47 @@ def gsd_rdf(
             system = (box, A_pos)
             aq = freud.locality.AABBQuery.from_system(system)
             nlist = aq.query(
-                B_pos, {"r_max": r_max, "exclude_ii": exclude_ii}
+                B_pos, {"r_max":r_max, "exclude_ii":exclude_ii}
             ).toNeighborList()
 
             if exclude_bonded:
                 pre_filter = len(nlist)
                 nlist.filter(
-                    molecules_A[nlist.point_indices]
+                    molecules_A[nlist.point.indices]
                     != molecules_B[nlist.query_point_indices]
                 )
                 post_filter = len(nlist)
 
             rdf.compute(aq, neighbors=nlist, reset=False)
+    normalization = post_filter / pre_filter if exclude_bonded else 1
+    return rdf, normalization
+
+def get_centers(gsdfile, new_gsdfile):
+    """Calculates the centers of a trajectory.
 
-        normalization = post_filter / pre_filter if exclude_bonded else 1
-        return rdf, normalization
+    This function calculates the centers of a trajetory given a GSD file
+    and stores them into a new GSD file just for centers. The new GSD file will be created in the same directory where the GSD file of trajectory is located. By default it will calculate the centers of an entire trajectory.
```

Comment:
```suggestion
    and stores them into a new GSD file just for centers. By default it will calculate the centers of an entire trajectory.
```

> The new GSD file will be created in the same directory where the GSD file of trajectory is located.

I am not sure if this is true. The path to new_gsdfile and gsdfile can be anything, and no logic guarantees they're in the same directory.